airbyte-metrics/reporter/src/main/java/io/airbyte/metrics/reporter/MetricRepository.java

```diff
@@ -25,6 +25,11 @@ class MetricRepository {
 
   private final DSLContext ctx;
 
+  // We have to report gauge metric with value 0 if they are not showing up in the DB,
+  // otherwise datadog will use previous reported value.
```

Comment:
nit: can we also add something stating that the alternative is longer and less readable sql queries?